beacon_node/lighthouse_network/gossipsub/src/behaviour.rs

```diff
@@ -3161,6 +3165,7 @@ where
         peer_id: PeerId,
         _: &Multiaddr,
         _: Endpoint,
+        _: PortUse,
```

Comment:
Is this related to `SO_REUSEADDR`? From the docs it seems more like a port-reuse option within libp2p itself, but I'm not sure.

The test I was trying to fix was the simulator, which currently crashes if you run it in a loop because the ports aren't fully freed by the time it runs again, and port-reuse is not enabled:

```fish
while [ $status -eq 0 ]
    cargo run --release --bin simulator fallback-sim
end
```

If we can get this test to run indefinitely without crashing, I'll be happy.

See also:

- https://github.com/sigp/lighthouse/pull/6239